pyanaconda/core/constants.py

```diff
@@ -424,6 +424,7 @@ class DisplayModes(Enum):
 SOURCE_TYPE_CDROM = "CDROM"
 SOURCE_TYPE_CLOSEST_MIRROR = "CLOSEST_MIRROR"
 SOURCE_TYPE_REPO_FILES = "REPO_FILES"
+SOURCE_TYPE_RPM_MOUNT = "RPM_MOUNT"
```

Comment:
I don't like the name at all. It is confusing, because it looks like we are trying to mount RPMs. Based on the code, it looks like we don't actually care if it is a mount point, do we? What about `REPO_PATH` or just `DIR`?

Reply:
`REPO_PATH` seems better to me. The `DIR` name is too generic.